contracts/fungible-token-full.aes

```diff
@@ -18,12 +18,13 @@
 // THIS IS NOT SECURITY AUDITED
 // DO NEVER USE THIS WITHOUT SECURITY AUDIT FIRST
 
-@compiler >= 4
+@compiler >= 6
 
 include "Option.aes"
+include "String.aes"
 
 /// @title - Fungible token with all the extensions - burn, mint, allowances
-contract FungibleTokenFull =
+main contract FungibleTokenFull =
```

Comment:
why main here? we can do without main and use compiler >= 5

Reply:
Sure.